### DISCUSSION TITLE
Guidance for PR Reviews with large cache changes

### DISCUSSION THREAD

**Discussion by dannyrb** [yarnpkg/berry#2859] [Q&A]
👋 Hello,

I'm using the Zero Install feature, but it is making pull request reviews difficult for our team. Is anyone else butting up against this issue? Is there an easy fix/workaround that we're not aware of?

1. Install one or more packages as a part of a new feature
2. Check in the `.yarn/` changes
3. Cache changes overwhelm/hide the one or two files that would normally be reviewed

<img width="730" alt="Screen Shot 2021-05-09 at 12 23 34 AM" src="https://user-images.githubusercontent.com/5797588/117560566-a6a41280-b043-11eb-90db-97922ada5b19.png">

If there is a better place to ask, any help to point me in the right direction would be appreciated 🙏

**Comment by noahnu** [comment]
You can filter out the zip files using File filter in github.

**Reply by osdiab** [reply]
filtering htem out just collapses them, but the individual file entries still get rendered, which usually means there's still hundreds of cache files to scroll past. And if there are too many cache files, even the file navigation box says that it has too many files, so doesn't work. it sucks - but i don't really see a better way besides not committing them until after code review, or making a separate branch that just contains the yarn cache updates, without intervention from Github to make it better.

**Comment by osdiab** [comment]
I experience the same thing, commenting to bump this if anyone has any good suggestions. I suspect (as in the other comment I wrote here) that it's not really great until Github makes some edits to their UI, though.
